src/pages/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Layout from './../components/Layout';
+import Hero from './../components/Hero';
+import Profile from './../components/Profile';
+import About from './../components/About';
+import Jobs from './../components/Jobs';
+import Contact from './../components/Contact';
```

Comment:
+1 on using absolute paths for these or even having an index file to import them all in one import. e.g. 
```typescript
import { Layout, Hero, Profile, About, Jobs, Contact } from 'components';
```